config/sources/families/rk35xx.conf

```diff
@@ -10,8 +10,8 @@
 source "${BASH_SOURCE%/*}/include/rockchip64_common.inc"
 
 BOOTSOURCE='https://github.com/radxa/u-boot.git'
-BOOTBRANCH='branch:stable-4.19-rock3'
-BOOTPATCHDIR="legacy"
+BOOTBRANCH='branch:next-dev-v2024.03'
+BOOTPATCHDIR="legacy/u-boot-radxa-rk3588"
```

Comment:
Stable-4.19-rock3 is for rk356x, so this rk3588 name is not correct. It's better to use rk35xx.

Reply:
Yes renaming it to rk35xx makes sense.